### DISCUSSION TITLE
U Ubuntu os problem : dev/sda5: clean

### DISCUSSION THREAD

**Discussion by Sir-Osama** [CloudBytes-Academy/CloudBytes.dev#60] [Q&A] [answered]
I was updating python 3 and while updating the installation some how has crushed I reboot my device and I got this : 
![IMG_20211011_201239_419](https://user-images.githubusercontent.com/86108333/136839656-62b62923-4e92-48ef-9f30-d24f34bce5df.jpg)
I look it up in ask Ubuntu community I tried the top answers and it didn't work.

**Comment by rehanhaider** [comment, marked as answer]
As far as I remember this is a graphics driver issue.

Try this solution https://stackoverflow.com/questions/68124511/booting-error-dev-sda2-clean-files-blocks

**Reply by rehanhaider** [reply]
Did you manage to fix this issue?

**Comment by jrumley** [comment]
Had the same issue. A bunch of programs were removed, including my wifi adapter driver. Once I connected to bluetooth and got it back I was able to use the stackoverflow link @rehanhaider posted. Still missing quite a few programs though.
